src/routes/Categories.js

```diff
@@ -1,7 +1,23 @@
-const Categories = () => (
-  <div className="categories">
-    <button type="submit">Check Status</button>
-  </div>
-);
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
+const Categories = () => {
+  const dispatch = useDispatch();
+  // const { categories } = useSelector((state) => state.categories);
+  const status = useSelector((state) => state.categories.status);
+
+  return (
+    <div className="categories">
+      <h1>{status}</h1>
+      <button
+        type="submit"
+        onClick={() => {
+          dispatch(checkStatus());
+        }}
+      >
+        Check Status
+      </button>
```

Comment:
- [ ] Clicking the `check status` button should dispatch the right action and display `under construction` on the page.